config/routes.rb

```diff
@@ -1,3 +1,10 @@
 Rails.application.routes.draw do
-   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  resources :users
+  devise_for :user
+  get 'home/index'
```

Comment:
Please, remove this route, we ain't going to use it! Same with `get 'user/index'` and `root to: "home#index"`